src/routes/safe/components/Apps/components/SecurityFeedbackModal/UnknownAppWarning.tsx

```diff
@@ -0,0 +1,57 @@
+import { useState } from 'react'
+import Box from '@material-ui/core/Box'
+import Checkbox from '@material-ui/core/Checkbox'
+import FormControlLabel from '@material-ui/core/FormControlLabel'
+import styled from 'styled-components'
+import { Text } from '@gnosis.pm/safe-react-components'
+import { StyledTitle } from './styles'
+
+type UnknownAppWarningProps = {
+  onHideWarning?: (hideWarning: boolean) => void
+}
+
+const UnknownAppWarning = ({ onHideWarning }: UnknownAppWarningProps): React.ReactElement => {
+  const [toggleHideWarning, setToggleHideWarning] = useState(false)
+
+  const handleToggleWarningPreference = (): void => {
+    onHideWarning?.(!toggleHideWarning)
+    setToggleHideWarning(!toggleHideWarning)
+  }
+
+  return (
+    <Box flexDirection="column">
+      <StyledTitle size="sm" color="error" bold centered={!!onHideWarning}>
+        ⚠️Warning!
+      </StyledTitle>
+      <Text size="xl" color="error">
+        <b>The application you are trying to use is not in our default list</b>
+      </Text>
+      <br />
+      <Text size="lg">Check the link you are using and ensure it comes from a trusted source</Text>
```

Comment:
Should we display the used URL here? So the user could double check it easier?
If we wanna help the user a bit we could also parse the URL and display the host separately to help prevent phishing attacks as URLs can look non suspicious on first looks.

i.e.
URL: `https://cloudflare-ipfs.com:ipfs%2FQme49gESuwpSvwANmEqo34yfCkzyQehooJ5yL7aHmKJnpZ@evil.com`
Hostname: `evil.com`